File: bddtests/src/main/kotlin/mocking/tpp/models/RequestPatientRecordReply.kt
package mocking.tpp.models

import javax.xml.bind.annotation.XmlAccessType
import javax.xml.bind.annotation.XmlAccessorType
import javax.xml.bind.annotation.XmlAttribute
import javax.xml.bind.annotation.XmlElement
import javax.xml.bind.annotation.XmlRootElement

@XmlRootElement(name = "RequestPatientRecordReply")
@XmlAccessorType(XmlAccessType.FIELD)
data class RequestPatientRecordReply(
        @XmlAttribute var patientId: String = "default patientId",
        @XmlAttribute var onlineUserId: String = "default onlineUserId",
        @XmlAttribute var uuid: String = "default uuid",
        @field:XmlElement(name="Event")
        @XmlElement var event: MutableCollection<Event> = mutableListOf()
)

File: xamarinintegrationtests/NHSOnline.HttpMocks/Tpp/Models/RequestPatientRecordReply.cs
using System.Collections.ObjectModel;
using System.Xml.Serialization;

namespace NHSOnline.HttpMocks.Tpp.Models
{
    public class RequestPatientRecordReply
    {
        [XmlElement("Event")]
        public Collection<Event>? Events { get; set; }
    }
}